src/common/geo/io/wkt/wkt_parser.yy

```diff
@@ -0,0 +1,198 @@
+%language "C++"
+%skeleton "lalr1.cc"
+%no-lines
+%locations
+%define api.namespace { nebula }
+%define parser_class_name { WKTParser }
+%lex-param { nebula::WKTScanner& scanner }
+%parse-param { nebula::WKTScanner& scanner }
+%parse-param { std::string &errmsg }
+%parse-param { nebula::Geometry** geom }
+
+%code requires {
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <cstddef>
+#include "common/geo/io/Geometry.h"
+
+namespace nebula {
+
+class WKTScanner;
+
+}
+
+}
+
+%code {
+    #include "WKTScanner.h"
+    static int yylex(nebula::WKTParser::semantic_type* yylval,
+                     nebula::WKTParser::location_type *yylloc,
+                     nebula::WKTScanner& scanner);
+}
+
+%union {
+    double                                  doubleVal;
+    Geometry*                               geomVal;
+    Point*                                  pointVal;
+    LineString*                             lineVal;
+    Polygon*                                polygonVal;
+    Coordinate*                             coordVal;
+    std::vector<Coordinate>*                coordListVal;
+    std::vector<std::vector<Coordinate>>*   coordListListVal;
+}
+
+/* destructors */
+%destructor {} <geomVal> <pointVal> <lineVal> <polygonVal>
+%destructor {} <coordVal> <coordListVal> <coordListListVal>
+%destructor {} <doubleVal>
+
+/* wkt shape type prefix */
+%token KW_POINT KW_LINESTRING KW_POLYGON
+
+/* symbols */
+%token L_PAREN R_PAREN COMMA
+
+/* token type specification */
+%token <doubleVal> DOUBLE
+
+%type <geomVal> geometry
+%type <pointVal> point
+%type <lineVal> linestring
+%type <polygonVal> polygon
+%type <coordVal> coordinate
+%type <coordListVal> coordinate_list
+%type <coordListListVal> coordinate_list_list
+
+%define api.prefix {wkt}
+
+%start geometry
+
+%%
+
+geometry
+  : point {
+    $$ = $1;
+    LOG(INFO) << "jie test" << static_cast<Point*>($$)->coord.x << static_cast<Point*>($$)->coord.y;
```

Comment:
what's this?

Reply:
Sorry, it's just for debug, I will delete it.